com.github.ransome1.sleek.yml

```
@@ -36,13 +31,24 @@ modules:
     sources:
       - type: git
         url: https://github.com/ransome1/sleek.git
-        commit: 5cbbbc64983a8140f3b7d227e7ef5fe4c7b09253
-      - type: inline
-        contents: |
-          yarn-offline-mirror /run/build/sleek/flatpak-node/yarn-mirror
-        dest-filename: .yarnrc
-      - generated-sources.json
+        commit: 988579f6c789f8050331259f455b5d38b4a23b6a
       - type: script
+        dest-filename: sleek-entrypoint.sh
         commands:
-          - exec env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/sleek/sleek
-        dest-filename: sleek.sh
\ No newline at end of file
+          - |
+            cat << 'EOF' > sleek-entrypoint.sh
+            #!/bin/sh
+            exec env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/opt/sleek/sleek "$@"
+            EOF
```

It works with just that:

```suggestion
          - exec env TMPDIR=$XDG_CACHE_HOME zypak-wrapper /app/opt/sleek/sleek "$@"
```